guides/common/modules/ref_required-red-hat-repositories.adoc

```diff
@@ -0,0 +1,52 @@
+[id="required-red-hat-repositories"]
+= Required Red Hat repositories
+
+You need the following repositories to manage hosts with {RHEL}.
+
+For {RHEL} 10 hosts::
+You require both:
+
+* *{RHEL} 10 for x86_64 - BaseOS (RPMs)*
+* *{RHEL} 10 for x86_64 - AppStream (RPMs)*
+
+For {RHEL} 9 hosts::
+You require both:
+
+* *{RHEL} 9 for x86_64 - BaseOS (RPMs)*
+* *{RHEL} 9 for x86_64 - AppStream (RPMs)*
+
+For {RHEL} 8 hosts::
+You require both:
+
+* *{RHEL} 8 for x86_64 - BaseOS (RPMs)*
+* *{RHEL} 8 for x86_64 - AppStream (RPMs)*
+
+For {RHEL} 7 hosts::
+You require the *{RHEL} 7 Server (RPMs)* repository.
++
+The difference between associating {RHEL} operating system release version with either *7Server* repositories or *7._y_* repositories is that *7Server* repositories contain all the latest updates while *{RHEL} 7._y_* repositories stop getting updates after the next minor version release.
+
+.Additional repositories for provisioning
```

Comment:
Since the repos are the same for RHEL-{8,9,10}, do we have to mention them specifically?